pam/integration-tests/integration_test.go

```diff
@@ -18,17 +18,20 @@ import (
 
 var daemonPath string
 
-func TestCLIIntegration(t *testing.T) {
+func TestCLIAuthenticate(t *testing.T) {
```

Comment:
This could be in another PR to decrease the number of changes on this one, but since I was already touching the integration tests and these diffs look a lot but are quite minimal, I added them on this PR. Let me know if you prefer me to split it.

Reply:
That’s perfect in my eyes.